src/components/implementation/no_interface/scheddev/sl_mod_fprr.c

```diff
@@ -7,6 +7,8 @@
 #define SL_FPRR_HIGHEST 0
 #define SL_FPRR_LOWEST  (SL_FPRR_NPRIOS-1)
 
+#define SL_FPRR_US_MIN  1000
```

Comment:
Kk. I'll make the names explicit here and for lets say `SL_FPRR_PERIOD_US_MIN`, I could use `SL_PERIOD_US`. 
I'm also going to change the PRIO related defines to `SL_FPRR_PRIO_HIGHEST` and `SL_FPRR_PRIO_LOWEST` for better clarity.